### DISCUSSION TITLE
[Feature request] Scrollbar

### DISCUSSION THREAD

**Discussion by ghost** [PaulWoitaschek/Voice#1360] [Ideas]
It would be useful to have a scrollbar to see where you are in the list of Audiobooks, especially when you have a big library

**Comment by versavius** [comment]
I have a large library on my phone at the moment (about 20 audiobooks), im just lazy and don't delete them when im done with them !

I dont like the idea of a scrollbar, sorry jony0008, i think it will clutter the interface, and since we scroll up and down with our fingers not necessary.

**Comment by ghost** [comment]
@sandybeachit IMO many have more than 20 audiobooks and I think not having to scroll up and down in a library with 20 audiobooks would help. For me the scrollbar would make a big difference because when scrolling now it is a big pain for me. The scrollbar don't need to be big, just a small one that can blend together with the interface.

**Comment by versavius** [comment]
But what extra functionality does the scrollbar bring that can't be done 
with just scrolling with your finger?

It's just the scrollbar won't be providing any functionality as we don't 
use a mouse on android to navigate windows.

I'm just thinking of Paul's original vision which is simplicity which I 
agree with.

You do realise with a scrollbar your still be scrolling up and down? So I 
don't see the benefit gained by adding a scrollbar unless I am missing 
something?

**Comment by ghost** [comment]
I like the vision of the complicity but I don't think it should be at the cost of good functionality. With a scrollbar you can just click on it and it will move you to that part of the cover view instantly without having to scroll with your finger.

**Comment by ghost** [comment]
@PaulWoitaschek Please make the scrollbar more responsive. Now when I click on it, it will only move to the end or beginning of the list depending on if I click on the lower or top part of it so it is not very helpful.

**Comment by PaulWoitaschek** [comment]
You can drag it.

**Comment by ghost** [comment]
@PaulWoitaschek It is the same thing, when I drag it to the lower part of the scrollbar it will move to the end (bottom) of the list and does not stay to where I dragged it. The same thing happens if I drag the scrollbar to the upper part of the scrollbar but then it will move to the top. I want it to stay where I dragged it and not move the view to the bottom or the top.

**Comment by versavius** [comment]
I don't get that issue if I drag the screen up and down it works fine. Scrollbars are really.bot meant for mobile devices (phones/tablets) so should only be on a desktop device.

There could be a possible bug here. Are you saying you are unable to move the screen up and down by dragging it? As the screen should not them move by itself?

**Comment by ghost** [comment]
@sandybeachit Dragging the screen up and down works when you only have a few books in your library but does not work when you have many books and want to easily browse your library to find one. Scrollbars are and should be used where they are useful and needed, it don't matter if it is on a mobile device or a desktop device. The scrollbar in Material Audiobook Player is very minimalistic.

**Comment by ghost** [comment]
@PaulWoitaschek Please add a search function or scroll bar. Without it your app becomes useless to me.
